File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1983.py
n = int(input())
maior = [float(j) for j in input().split(' ')]
for i in range(n-1):
    v = [float(j) for j in input().split(' ')]
    if(v[1]>maior[1]):
        maior[1]=v[1]
        maior[0]=v[0]
if(maior[1]>=8):
    print(int(maior[0]))
else:
    print("Minimum note not reached")

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1983.cpp
#include <iostream>

using namespace std;

int main(){
    double n, maior[2], v, k;
    cin >> n >> maior[0] >> maior[1];
    for(int i=1;i<n;i++){
        cin >> v >> k;
        if(k>maior[1]){
            maior[0]=v;
            maior[1]=k;
        }
    }
    if(maior[1]>=8){
        cout << maior[0] << endl;
    } else{
        cout << "Minimum note not reached" << endl;
    }
    
    return 0;
}